t/get_adjusted.t

```diff
@@ -29,9 +29,19 @@ my @tests = (
         {aa=>0.1, bb=>0.1},
         {aa=>0}
     ],
+    [ { type => 'fruit' }, "apple", "apple" ],
```

Comment:
Such usage of 'type' breaks json-schema specification (it allows just closed list of types).
Maybe it's better to rely on 'format' value?
`{ type => 'string', format => 'fruit' }`

Reply:
Hi. I confess I tried lookup for the Json-Schema specification, but found different document. We are basing on this specification? https://json-schema.org/

If so, I will read it and try to understand what you are suggesting. Thanks!

Reply:
Hi! These are 'type' and 'format' sections of current draft
https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.1.1
https://json-schema.org/latest/json-schema-validation.html#rfc.section.7